.github/workflows/prometheus.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: Prometheus check
+
+on:
+  pull_request #:
+    # types: [labeled, opened, synchronize, reopened]
+
+jobs:
+  check:
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    # if: |
```

Comment:
Is this commented code for future PRs or similar?

Reply:
I need to get the labels created... The PR should really still be a draft but draft PRs don’t get reviewed.